model/device/registry.yaml

```
@@ -3,23 +3,33 @@ groups:
     type: attribute_group
     display_name: Device Attributes
     brief: >
-        Describes device attributes.
+      Describes device attributes.
     attributes:
       - id: device.id
         type: string
         stability: development
         brief: >
           A unique identifier representing the device
-        note: >
-          The device identifier MUST only be defined using the values outlined below. This value is not an advertising
-          identifier and MUST NOT be used as such.
-          On iOS (Swift or Objective-C), this value MUST be equal to the [vendor identifier](https://developer.apple.com/documentation/uikit/uidevice/1620059-identifierforvendor).
-          On Android (Java or Kotlin), this value MUST be equal to the Firebase Installation ID or a globally unique
-          UUID which is persisted across sessions in your application. More information can be found [here](https://developer.android.com/training/articles/user-data-ids)
-          on best practices and exact implementation details.
-          Caution should be taken when storing personal data or anything which can identify a user. GDPR and
-          data protection laws may apply, ensure you do your own due diligence.
-        examples: ['2ab2916d-a51f-4ac8-80ee-45ac31a28092']
+        note: |
+          Its value SHOULD be identical for all apps on a device and it SHOULD NOT change if an app is uninstalled and re-installed.
+          However, it might be resettable by the user for all apps on a device.
+          Hardware IDs (e.g. vendor-specific serial number, IMEI or MAC address) MIGHT be used as values.
```

(just the more common normative word used in the spec)
```suggestion
          Hardware IDs (e.g. vendor-specific serial number, IMEI or MAC address) MAY be used as values.
```